test/Microsoft.AspNetCore.ResponseCaching.Tests/ResponseCachePolicyProviderTests.cs

```diff
@@ -11,26 +11,47 @@ namespace Microsoft.AspNetCore.ResponseCaching.Tests
 {
     public class ResponseCachePolicyProviderTests
     {
+        public static TheoryData<string> CacheableMethods
+        {
+            get
+            {
+                return new TheoryData<string>
+                {
+                    HttpMethods.Get,
+                    HttpMethods.Head
+                };
+            }
+        }
+
         [Theory]
-        [InlineData("GET")]
-        [InlineData("HEAD")]
+        [MemberData(nameof(CacheableMethods))]
```

Comment:
Nope, the compiler complains that ` error CS0182: An attribute argument must be a constant expression, typeof expression or array creation expression of an attribute parameter type`. The reason being that the definition of these methods are `static readonly` not `const`.